swan-lake/resources/featured-scenarios/call-llms-with-natural-expressions.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: ballerina-calling-llms-with-natural-expressions-left-nav-pages-swanlake
+title: Call Large Language Models (LLMs) with Natural Expressions 
+description: Bring AI-powered capabilities into your integrations using natural expressions to call LLMs with automatic binding of the response to user-defined types.
+keywords: ballerina, AI, LLM, large language model
+permalink: /learn/call-llms-with-natural-expressions/
+active: call-llms-with-natural-expressions
+intro: This guide helps you understand the basics of Ballerina natural expressions.
+---
+
+## Set up the prerequisites
+
+To complete this tutorial, you need:
+
+1. Ballerina 2201.13.0-m1 (Swan Lake) or greater. Install [Ballerina](/downloads/) and use the `bal dist pull` command to pull the milestone version.
+
+    ```
+    bal dist pull 2201.13.0-m1
+    ```
+
+2. A text editor
+  >**Tip:** Preferably, <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a> with the 
+  <a href="https://wso2.com/ballerina/vscode/docs/" target="_blank">Ballerina extension</a> installed. Enable [experimental features](/learn/vs-code-extension/configure-the-extension/) for Ballerina.
+
+3. A command terminal
+
+## Understand the implementation
+
+This tutorial describes how to use Ballerina's natural expressions to integrate calling large language models (LLMs). In this sample you will develop a REST API with a single GET resource that accepts the name of a country, the interests of the user, and a count and uses a natural expression to call an LLM to identify the specified number of attractions in the specified country that may be of interest to the user. The expected structure of the response is indicated by the expected type and natural expressions automatically bind the response to the expected type.
+
+## Create the service package
+
+Ballerina uses packages to group code. You need to create a Ballerina package and write the business logic in it. In the terminal, execute the command below to create the Ballerina package for the implementation.
+
+> **Info:** For more information on Ballerina packages, see [Organize Ballerina code](/learn/organize-ballerina-code/).
+
+```
+$ bal new attraction_finder_service
+``` 
+
+This creates a directory named `attraction_finder_service` with the files below.
```

Comment:
`devcontainer.json` and `gitignore` files also get created

Reply:
These are intentionally left out, check the other samples under this category.